### PR TITLE
K3s Cluster Initialization

### PR DESCRIPTION
**Pull request by johnworth**
Adds more documentation, a fix for getting the HAProxy node configured before the workers are set up, and installation of the following resources:

* namespaces
* service accounts
* roles
* role bindings
* cluster role bindings

### CODE REVIEW

File: ansible/kubernetes/README.md
Comment (ianmcorvidae) [inline ansible/kubernetes/README.md:63]: If we're changing this here, we should change it in the haproxy playbooks (for the DE haproxy) as well. Unless that playbook shouldn't use these groups, but it seems like unnecessary duplication not to.

**Review by ianmcorvidae — COMMENTED** [review]
Didn't spot anything that seemed off to me except a small update to another role. I'm also fine to make that change after you merge this one if you'd prefer, since the other role is my work!

**Review by slr71 — APPROVED** [review]
Looks good to me :shipit:

File: ansible/kubernetes/README.md
Comment (johnworth) [reply ansible/kubernetes/README.md:63]: Good catch, thanks. I'll look into getting it changed as a part of this PR.

Comment (johnworth) [reply ansible/kubernetes/README.md:63]: Should be done now, let me know if I missed anything else.

**Pull request merged**
